src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -241,18 +236,18 @@ public HyperwalletBusinessStakeholder updateBusinessStakeholder(String userToken
      *
      * @param userToken                String
      * @param businessStakeholderToken Hyperwallet Stakeholder representation
-     * @param multiPart                FormdataMultipart to get uploaded
+     * @param uploadData                HyperwalletVerificationDocument to get uploaded
      * @return HyperwalletBusinessStakeholder updated Stakeholder with document status
      */
-    public HyperwalletBusinessStakeholder uploadDocumentBusinessStakeholder(String userToken, String businessStakeholderToken,
-                                                                            FormDataMultiPart multiPart) {
+    public HyperwalletBusinessStakeholder uploadStakeholderDocuments(String userToken, String businessStakeholderToken,
+                                                                     HyperwalletVerificationDocument uploadData) {
```

Comment:
I believe we discussed that it should be a list of documents

```suggestion
    public HyperwalletBusinessStakeholder uploadStakeholderDocuments(String userToken, String businessStakeholderToken,
                                                                     List<HyperwalletVerificationDocument> verificationDocumentList) {
```